### PR TITLE
Enable Audius

### PR DESCRIPTION
**Pull request by reubenr0d**

### CODE REVIEW

File: packages/shared/src/data/stakers.tsx [hunk 1]
```diff
@@ -71,13 +71,13 @@ export const stakers: Record<string, Staker> = {
...
-    available: false,
+    available: true,
     logo: "AUDIO.svg",
     bwLogo: "AUDIO.svg",
     bwTenderLogo: "tenderAUDIO.svg",
     neonLogo: "landing/neon-audio.png",
     symbol: "AUDIO",
     chainId: getChainId(ChainId.Mainnet, ChainId.Rinkeby),
-    hasPermit: false,
+    hasPermit: true,
```
Comment (vmaark) [inline packages/shared/src/data/stakers.tsx:81]: Cool!

**Review by vmaark — APPROVED** [review]
👍

**Review by kyriediculous — CHANGES_REQUESTED** [review]
There's a merge conflict in this, good spot that AUDIO has Permit support. 

Let's resolve the merge conflict , after that GTG 🚢

**Review by vmaark — APPROVED** [review]
👍

**Review by kyriediculous — APPROVED** [review]
LGTM 👍

**Pull request merged**